template/koa-rest-api/src/config.ts

```diff
@@ -32,7 +32,7 @@ const configs: Record<Environment, () => Omit<Config, 'environment'>> = {
   }),
 
   test: () => ({
-    logLevel: Env.string('LOG_LEVEL', { default: 'silent' }),
+    logLevel: Env.string('LOG_LEVEL', { default: 'debug' }),
```

Comment:
Is it worth just 

```suggestion
    logLevel: 'debug',
```

cos you want the tests not to depend on env too much yeah?